src/azul/__init__.py

```diff
@@ -680,6 +680,13 @@ def portal_db_bucket(self) -> str:
     def portal_db_object_key(self) -> str:
         return f'azul/{self.deployment_stage}/portals/{self.dss_deployment_stage(self.dss_endpoint)}-db.json'
 
+    @property
+    def layer_bucket(self) -> str:
+        return self.versioned_bucket
+
+    def layer_object_key(self, requirements_hash) -> str:
```

Comment:
It's funny how a small method like this can still fail to cleanly separate concerns.